### DISCUSSION TITLE
如何更改页眉文字、标题页logo与文字、去掉英文版面？

### DISCUSSION THREAD

**Discussion by spoilvoid** [sjtug/SJTUThesis#981] [提问]
想要用作其它课程用途，但是发现有些部分不会修改
1.标题页logo与文字如何自定义修改？
![屏幕截图 2023-10-06 212514](https://github.com/sjtug/SJTUThesis/assets/66478971/af12c687-474f-451e-9d0c-78d1aa497fda)
2.页眉文字如何修改？
![屏幕截图 2023-10-06 212644](https://github.com/sjtug/SJTUThesis/assets/66478971/0a68c5c1-6012-4da6-89da-ed4a0526a43d)
3.如何删去英文标题页？

求解答！！！

**Comment by LogCreative** [comment]
- logo 修改参照 https://github.com/sjtug/SJTUThesis/discussions/911#discussioncomment-5500215
- logo 文字与页眉需要修改主题，https://github.com/sjtug/SJTUThesis/discussions/938#discussioncomment-5895956
- 删去英文标题页，目前比较直接的方法是修改下面的代码
https://github.com/sjtug/SJTUThesis/blob/2cbbecf929649aba722cd7f3d7fcafc52aec2d51/texmf/tex/latex/sjtutex/sjtuthesis.cls#L1089-L1090

为

```latex
\UseInstance { sjtu } { title / zh }
```
